components/drafts/panel/panel_body.tsx

```diff
@@ -68,6 +76,27 @@ function PanelBody({
                 <div className='DraftPanelBody__right'>
                     <div className='post__header'>
                         <strong>{displayName}</strong>
+                        {priority && (
+                            <div className='DraftPanelBody__priority'>
+                                {priority?.priority && (
+                                    <PriorityLabel
+                                        size='xs'
+                                        priority={priority.priority}
+                                    />
+                                )}
+                                {priority?.requested_ack && (
+                                    <div className='DraftPanelBody__priority-ack'>
+                                        <CheckCircleOutlineIcon size={14}/>
+                                        {!priority.priority && (
+                                            <FormattedMessage
+                                                id={'post_priority.request_acknowledgement'}
+                                                defaultMessage={'Request acknowledgement'}
+                                            />
+                                        )}
+                                    </div>
+                                )}
+                            </div>
+                        )}
```

Comment:
the optional chaining operators should not be needed since you already check the existence in the first condition. Is `tsc` not getting this? If not probably move this to a var and wrap it in an if condition?

It just feels a bit awkward that there are optional chaining operators followed by a non-null assertion operator 🤓

Reply:
oh, I might have forgotten to update those